<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialPort</class>
 <widget class="QMainWindow" name="serialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_parityBit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_serialNum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dataBit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_stopBit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_dataBit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_parityBit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_serialNum">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_stopBit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_baud">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>57600</string>
    </property>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_baud">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openSerial">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clean">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>清空消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="LED">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:0.1 rgba(255, 255, 255, 255), stop:0.2 rgba(255, 176, 176, 167), stop:0.3 rgba(255, 151, 151, 92), stop:0.4 rgba(255, 125, 125, 51), stop:0.5 rgba(255, 76, 76, 205), stop:0.52 rgba(255, 76, 76, 205), stop:0.6 rgba(255, 180, 180, 84), stop:1 rgba(255, 255, 255, 0));

border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
